scripts/requirements.txt

```diff
@@ -1,5 +1,5 @@
-breathe==4.9.1
-sphinx==1.7.5
-docutils==0.14
+breathe>=4.9.1
+sphinx>=1.7.5
+docutils>=0.14
```

Comment:
I just noticed Travis relies on this file too. So this change affects CI too, making it less deterministic? If that's considered an issue then maybe we could have two different `requirements.txt` files?

Reply:
@xiulipan any comments for travis here ?